Assets/World/Units/Unit2D.gd

```diff
@@ -7,15 +7,20 @@ class_name Unit2D
 
 @onready var person_sprite: AnimatedSprite2D = self.get_child(0)
 
+enum is_full {
+  full     = 0,
+  not_full = 1,
+}
+
```

Comment:
create a var of the corresponding type:
var carrier_state := CarrierState.Empty